src/views/organizations/OrganizationDetailView.vue

```diff
@@ -111,13 +122,17 @@ watchEffect(() => {
     <div v-if="!datasets?.length">
       Pas de jeu de données pour cette organisation.
     </div>
-    <ul v-else class="fr-grid-row fr-grid-row--gutters es__tiles__list">
-      <li v-for="d in datasets" :key="d.id" class="fr-col-12 fr-col-lg-4">
-        <Tile
-          :link="`/datasets/${d.slug}`"
-          :title="d.title"
-          :description="d.description"
-          :img="d.organization?.logo"
+    <ul v-else class="fr-grid-row fr-grid-row--gutters">
```

Comment:
tu peux ajouter le role sur la liste pour te débarasser des puces.

```suggestion
    <ul v-else class="fr-grid-row fr-grid-row--gutters" role="list">
```